internal/worker/export_test.go

```diff
@@ -0,0 +1,13 @@
+package worker
+
+import "os/exec"
+
+// MockExecCommand replaces the exec.Command() wrapper and returns a function
+// that can be called to restore the original.
+func MockExecCommand(mock func(name string, arg ...string) *exec.Cmd) func() {
```

Comment:
Nitpick: we tend to name the `restore` function value, to make the purpose of the function self-explanatory (i.e. https://github.com/osbuild/osbuild-composer/blob/c6f40834d38c46c289ad2ff54cac7e735f644e89/internal/cloudapi/v2/export_test.go#L14).